src/ec/packages/ec-component-site-header/site-header.html.twig

```diff
@@ -117,24 +112,24 @@
   <div class="ecl-site-header__container ecl-container">
     <div class="ecl-site-header__banner">
       <a 
-        class="ecl-link ecl-link--standalone" 
-        href="{{ _header_link.url }}" 
-        aria-label="{{ _header_link.aria_label }}"
+        class="ecl-link ecl-link--standalone"
+        href="{{ _logo.url }}"
+        aria-label="{{ _logo.title }}"
       >
         <img 
-          alt="{{ _header_image.alt }}" 
-          title="{{ _header_image.title }}" 
-          class="ecl-site-header__logo-image" 
-          src="{{ _header_image.src }}"
+          alt="{{ _logo.alt }}"
+          title="{{ _logo.title }}"
+          class="ecl-site-header__logo-image"
+          src="{{ _logo.src }}"
         />
       </a>
       <div class="ecl-site-header__selector">
         <a 
-          class="ecl-link ecl-link--standalone" 
-          href="{{ _language.url }}"
-          data-ecl-language-selector
+          class="ecl-link ecl-link--standalone ecl-site-header__selector-link"
+          href="{{ _language_selector.url }}"
```

Comment:
ok that we might have existing inconsistencies, but let's avoid introducing new ones. :)
At the top of the page, in the specs, you have this:
```
- "language_selector" (associative array) (default: predefined structure): Language switcher settings. format:
        {
          "href": (string) (default: ''): URL for switcher
```
So then i don't understand your concerns, yes we have to change it, both because like this we don't get any value from the specs we use from ecl and because we are saying: use "href" and then in the template we pass instead an unexisting url key.. ;)

Reply:
Sorry, I thought it was parameters for the language_selector items:
https://github.com/ec-europa/ecl-twig/pull/151/commits/e903b615b0c3d14d129dcbe208dd7f622dbcb797

Thanks, it's fixed now!